bin/concat_tsv.py

```diff
@@ -28,35 +30,39 @@ def parse_args(args=None):
     return parser.parse_args()
 
 
+# Peptide Predictions remain unsorted, otherwise full dataframe needs to be loaded to memory, which would be computational very intensive.
```

Comment:
You mean the peptides within the chunks before merging? Because the input channel for `MERGE_PREDICTIONS` you sorted

Reply:
Oh right, this can be deleted. I added the comment before actually solving the sorting.!

Reply:
```suggestion
```